sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -361,23 +365,19 @@ void gnrc_rpl_parent_update(gnrc_rpl_dodag_t *dodag, gnrc_rpl_parent_t *parent)
     }
 }
 
-int _compare_parents(gnrc_rpl_parent_t *p1, gnrc_rpl_parent_t *p2)
+static int _compare_parents(gnrc_rpl_parent_t *p1, gnrc_rpl_parent_t *p2)
 {
     return p1->dodag->instance->of->which_parent(p1, p2) == p1 ? -1 : 1;
 }
 
-gnrc_rpl_parent_t *gnrc_rpl_find_preferred_parent(gnrc_rpl_dodag_t *dodag)
+static gnrc_rpl_parent_t *_gnrc_rpl_find_preferred_parent(gnrc_rpl_dodag_t *dodag)
```

Comment:
could you migrate the former doxy here? 
``` C
 /**
 * @brief   Find the parent with the lowest rank and update the DODAG's preferred parent
 *
 * @param[in] dodag     Pointer to the DODAG
 *
 * @return  Pointer to the preferred parent, on success.
 * @return  NULL, otherwise.
 */
```
I know that this is not generated in, since this function is only local now  but I find it helpful to understand what happens under the hood